src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -998,6 +1000,9 @@ class Blocksmith extends React.Component {
     const currentBlock = getBlockForKey(contentState, anchorKey);
     const previousBlock = contentState.getBlockBefore(anchorKey);
     const nextBlock = contentState.getBlockAfter(anchorKey);
+    const firstLine = isOnFirstLineOfBlock(editorState, 'line-length-tester');
+    const lastLine = isOnLastLineOfBlock(editorState, 'line-length-tester');
```

Comment:
@nanetteCodes as you've seen, the process for determining this is pretty intensive and should not be done unless necessary - so defer these until we know for sure we need it